docs/standard/parallel-programming/snippets/vb/unwrap.vb

```diff
@@ -1,7 +1,7 @@
 ﻿Imports System.Threading
 
 Module UnwrapExample
-    Sub Main()
+    Sub Main2()
```

Comment:
Is that intended?

Reply:
Yes, it was needed to disambiguate having multiple `Sub Main`.

Reply:
@IEvangelist I think we can keep it as `Main` and fix the value of `StartupObject` to avoid the error.